python/nano/src/bigdl/nano/pytorch/inference/optimizer.py

```diff
@@ -163,6 +165,10 @@ def optimize(self, model: nn.Module,
                Default: False.
         :param latency_sample_num: (optional) a int represents the number of repetitions
                to calculate the average latency. The default value is 100.
+        :param includes: (optional) a list of acceleration methods that will be included in the
+               search. Default to None meaning including all available methods.
+        :param excludes: (optional) a list of acceleration methods that will be excluded from the
+               search.
```

Comment:
should we provide all the available choices here in the api doc?

Reply:
and of course the special logic to "original" since it has to be executed, so we may ignore it's presence in `includes` or `excludes`.

Reply:
> should we provide all the available choices here in the api doc?

I think we should provide all available choices name in the api doc.